packages/did/src/Did.chain.ts

```diff
@@ -42,6 +44,11 @@ import {
   parse,
 } from './Did.utils.js'
 
+import {linkedInfoFromChain, getDidUriFromKey,} from './index.js'
+import { Chain } from '@cord.network/network'
```

Comment:
This is an issue, as Chain itself depends on did if I am right. Or else, we should mention the dependency of network in did's package.json file (compared to few other packages)